src/constants/configs.ts

```diff
@@ -1,3 +1,6 @@
 const ROOT = 'EnvelopingConfig';
 
+export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
+export const ESTIMATED_GAS_CORRECTION_FACTOR = '1.0'; // TODO: if needed put a correction factor to mitigate RSK node gas miscalculation if execution includes refunds
+export const INTERNAL_TRANSACTION_ESTIMATE_CORRECTION = '20000'; // When estimating the gas an internal call is going to spend, we need to substract some gas inherent to send the parameters to the blockchain
```

Comment:
1. This file should probably only contain config-file-related constants, however I'm open to discussion.
3. If the value are numeric let's try to have them numeric, or big number, but not string

Reply:
I just moved this to utils.ts and changed it to numeric.